### ISSUE TITLE
Changing the phrase for preprint versions (ing.grid preprints)

### ISSUE THREAD

**Issue by mattiker** [BirkbeckCTP/janeway#4099]
The current phrase for preprints at ing.grid is _This is a Preprint and has not been peer reviewed. This is version {x} of this Preprint._ 
Example: https://preprints.inggrid.org/repository/view/14/
But some of the preprints are already reviewed and are linked to the published version.
Can you change the wording to: _This article is a preprint (current version: {x})_

**Comment by ajrbyers** [comment]
@mattiker has the DOI of the published preprint been added to the preprint's metadata?

**Comment by mattiker** [comment]
No, the preprints don't have a DOI. They will only get one after the open peer review when they are transferred to the journal.

**Comment by ajrbyers** [comment]
Just for clarity I'm talking about the Published DOI field. This field allows you to say "This Preprint has a published version and this is the DOI for it"

<img width="951" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/8321378/8eafc5ae-1b27-4c9b-8245-e5ff9d317df7">

If you set this to the article DOI the wording will change.

**Comment by mattiker** [comment]
Got it, I just changed it for https://preprints.inggrid.org/repository/view/14/ and the wording now is:
_This is a Preprint and has not been peer reviewed. **The published version of this Preprint is available: https://doi.org/10.48694/inggrid.3866**. This is version 8 of this Preprint._ 
But the first sentence is not quite right because the preprints actually are open peer reviewed. So the phrase _This article is a preprint (current version: {DOI})_ reflects this a bit better.

**Comment by mauromsl** [comment]
We want to carefully consider if there are any further implications for repositories if we were to remove this wording broadly. 
@alainna how would this affect the repositories that you run? The initial implementation of this page was designed to match EarthArxiv's previous platform IIRC.

**Comment by mattiker** [comment]
@mauromsl We like the idea from @alainna using tags to customize the phrases and would favor this solution if possible:

> Perhaps it might be useful for this to be a Repository Wizard template, which the repository could use `{{repository.object}}` tags to build for themselves? Ideally it would be able to fill in data from custom repository fields as well. For example, if the repository added a custom field for "peer review", then `{{repository.customfieldwhatever}}` would read that field and display "has not been peer reviewed", etc.
